pkg/mkrootfs-ext4/make-rootfs

```diff
@@ -41,11 +41,11 @@ IMGFILE=/rootfs.img
   ROOTFS_FILE_SIZE_KB=$(du -sk . | awk '{print $1}')
   ROOTFS_PART_BLOCKS=$(( $ROOTFS_FILE_SIZE_KB / 4 + $ROOTFS_PART_HEADROOM_BLOCKS ))
   ROOTFS_PART_SIZE=$(( $ROOTFS_PART_BLOCKS * $ROOTFS_BLOCKSZ ))
-  ROOTFS_PART_SIZE_KB=$(( ( ($ROOTFS_PART_SIZE + 1024) / 1024 ) / 1024 * 1024 ))
-  ROOTFS_PART_SECTORS=$(( $ROOTFS_PART_SIZE_KB * 2 ))
+  ROOTFS_PART_SIZE_KB=$(((ROOTFS_PART_SIZE + 1023) / 1024))
```

Comment:
`ROOTFS_PART_SIZE_KB=$(((ROOTFS_PART_SIZE + 1023) / 1024))` produces different results compared to `ROOTFS_PART_SIZE_KB=$(( ( ($ROOTFS_PART_SIZE + 1024) / 1024 ) / 1024 * 1024 ))`. Is this ok?

Reply:
```
#!/bin/bash

calc() {
        x=$1
        echo $(( ( (x + 1024) / 1024 ) / 1024 * 1024 ))
}

calc2() {
        x=$1
        echo $(((x + 1023) / 1024))
}

calc 1025*1024
calc2 1025*1024
```

Reply:
yes, different. because old implementation bogously reduces size, but we expect round up. when we provide exactly 1025kb, why the size of the resulting image is reduced to 1024kb?